assets/html/scss/documenter/layout/_search.scss

```diff
@@ -1,69 +1,133 @@
 // Additional styling for the search page
-#documenter .docs-main {
-  #documenter-search-info {
-    margin-bottom: 1rem;
-  }
-
-  #documenter-search-results {
-    list-style-type: circle;
-    list-style-position: outside;
-
-    li {
-      margin-left: 2rem;
-    }
-
-    .search-result-link {
-      border-radius: 0.7em;
-      transition: all 300ms;
-    }
-    
-    .search-result-link:hover, .search-result-link:focus {
-      background-color: $search-result-link-hover;
-    }
-    
-    .search-result-link .property-search-result-badge {
-      transition: all 300ms;
-    }
-    
-    .property-search-result-badge {
-      padding: 0.15em 0.5em;
-      font-size: 0.8em;
-      font-style: italic;
-      text-transform: none !important;
-      line-height: 1.5;
-      color: $search-result-badge-color;
-      background-color: $search-result-badge-background-color;
-      border-radius: 0.6rem;
-    }
-    
-    .search-result-link:hover .property-search-result-badge, .search-result-link:focus .property-search-result-badge {
-      color: $search-result-link-text-color;
-      background-color: $search-result-link-text-background-color;
-    }
-    
-    .search-result-highlight {
-      background-color: $search-result-highlight;
-      color: black;
-    }
-    
-    .search-divider {
-      border-bottom: 1px solid $border;
-    }
-    
-    .search-result-title {
-      color: $search-result-title-text-color;
-    }
-    
-    .w-100 {
-      width: 100%;
-    }
-    
-    .gap-2 {
-      gap: 0.5rem;
-    }
-    
-    .gap-4 {
-      gap: 1rem;
-    }
-  }
+/* Code from https://dylanatsmith.com/wrote/styling-the-kbd-element */
+kbd.search-modal-key-hints {
+  border-radius: 0.25rem;
+  border: 1px solid $search-modal-kbd-color;
+  box-shadow: 0 2px 0 1px $search-modal-kbd-color;
+  cursor: default;
+  font-size: 0.9rem;
+  line-height: 1.5;
+  min-width: 0.75rem;
+  text-align: center;
+  padding: 0.1rem 0.3rem;
+  position: relative;
+  top: -1px;
+}
+
+.search-min-width-50 {
+  min-width: 50%;
+}
+
+.search-min-height-100 {
+  min-height: 100%;
+}
+
+.search-modal-card-body {
+  max-height: 500px;
```

Comment:
I don't see why we wouldn't want to fill up pretty much the whole vertical space? I was thinking maybe we could do about `margin-top/bottom` instead of `max-height`, but this would work too:

```suggestion
  max-height: calc(100vh - 15rem);
```

![image](https://github.com/JuliaDocs/Documenter.jl/assets/147757/e78c3bc0-d1ee-4d43-80f7-1109921c22ab)
![image](https://github.com/JuliaDocs/Documenter.jl/assets/147757/ae886079-bb35-4dfa-bda6-396495d3b74a)